src/Raven.Server/Commercial/LicenseStatus.cs

```diff
@@ -105,6 +105,11 @@ public int MaxClusterSize
 
         public bool HasEncryption => GetValue<bool>("encryption");
 
+        public bool HasCompression =>
```

Comment:
We'll add the compression to all the licenses.
You also need to update `LicenseValidator`

Reply:
done